db/schema.rb

```diff
@@ -214,11 +215,13 @@
     t.string   "email"
     t.string   "uid"
     t.text     "permissions"
-    t.boolean  "remotely_signed_out", default: false
-    t.datetime "created_at",                          null: false
-    t.datetime "updated_at",                          null: false
+    t.boolean  "remotely_signed_out",     default: false
+    t.datetime "created_at",                              null: false
+    t.datetime "updated_at",                              null: false
     t.string   "organisation_slug"
-    t.boolean  "is_robot",            default: false
+    t.boolean  "is_robot",                default: false
+    t.string   "organisation_content_id"
```

Comment:
One last thing: the `organisation_content_id`and `disabled` lines should be the other way around - I get a schema diff in dev at the moment after running migrations.